docs/README.md

```diff
@@ -0,0 +1,60 @@
+# Vitessce Documentation
+
+This documentation website was built using [Docusaurus 2](https://v2.docusaurus.io/).
+
+## Prepare
+
+This documentation site depends on the Vitessce production library located in `../dist/`.
+
+```sh
+cd ..
+npm install
+npm run build-lib:prod
+npm link docs/node_modules/react
+cd docs
+```
+
+## Installation
+
+Install the docusaurus dependencies.
+
+```sh
+npm install
```

Comment:
I think these steps are in the wrong order since `docs/node_modules/react` doesn't necessarily exist.  Maybe make a note about the first time you do these steps if they are right?